app/_src/gateway-operator/customization/data-plane-image.md

```diff
@@ -2,6 +2,11 @@
 title: Customizing the Data Plane image
 ---
 
+{% assign gatewayConfigApiVersion = "v1beta1" %}
+{% if_version lte:1.1.x %}
+{% assign gatewayConfigApiVersion = "v1alpha1" %}
+{% endif_version %}
```

Comment:
Nit: can this be placed in a reusable import somewhere?

Reply:
A proposal that I sent to docs earlier today:

I'm using the following pattern more and more in the KIC docs as we migrate from alpha to beta versions of CRDs:
```
{% assign gatewayConfigApiVersion = "v1beta1" %}
{% if_version lte:1.1.x %}
{% assign gatewayConfigApiVersion = "v1alpha1" %}
{% endif_version %}
```

I've just had to add it to three different pages which feels wrong to me, so I've been thinking of alternatives.
What if we had variable configs somewhere in the site? Maybe something like this:

```
# app/_data/product_vars/gateway-operator.yaml

- name: gatewayConfigApiVersion
  default: v1beta1
  overrides:
    - constraint: "lte:1.1.x"
      value: v1alpha1
```

Then I could call `page.vars.gatewayConfigApiVersion` in the template (single source plugin would populate `page.vars` based on constraints) and know that it would just work.

Reply:
I've applied the change you suggested, but the version is not rendered at all. I suppose there is some piece missing, do you have any clue @mheap?

Reply:
Ah! This doesn't actually work. It's a proposal to solve the duplication. For now we have to do the `assign` in every file